app/controllers/carts_controller.rb

```diff
@@ -12,6 +12,12 @@ def create
     redirect_back(fallback_location: items_path)
   end
 
+  def update
+    item = Item.find(params[:item_id])
+    @cart.update_item(item.id, params[:Quantity])
```

Comment:
Love how this @cart instance variable is so easy to call. GOod job.

Reply:
@limsammy Yeah its a fantastic too. We were about to go to the controller before realizing there was no need.

Reply:
I think @cart might be too easy to call

Reply:
I think somebody tagged somebody instead of actually making a call for @cart in the comments. Haha. You just pulled this guy into our project. I pity him.